src/NzbDrone.Core/Indexers/Definitions/Redacted.cs

```diff
@@ -1,23 +1,56 @@
+using System;
 using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using FluentValidation;
 using NLog;
 using NzbDrone.Common.Http;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Annotations;
 using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers.Exceptions;
+using NzbDrone.Core.Indexers.Gazelle;
+using NzbDrone.Core.IndexerSearch.Definitions;
 using NzbDrone.Core.Messaging.Events;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
 
 namespace NzbDrone.Core.Indexers.Definitions
 {
-    public class Redacted : Gazelle.Gazelle
+    public class Redacted : TorrentIndexerBase<RedactedSettings>
```

Comment:
Changing the ConfigContract breaks existing instances, this needs a migration to avoid cast errors

![image](https://user-images.githubusercontent.com/376117/122852348-73110500-d2de-11eb-8450-5c201b524c3b.png)